src/topology/continuous_function/algebra.lean

```diff
@@ -52,6 +53,10 @@ instance has_mul [has_mul β] [has_continuous_mul β] : has_mul C(α, β) :=
 @[simp, norm_cast, to_additive]
 lemma coe_mul [has_mul β] [has_continuous_mul β] (f g : C(α, β)) : ⇑(f * g) = f * g := rfl
 
+@[simp, to_additive]
+lemma mul_apply [has_mul β] [has_continuous_mul β] (f g : C(α, β)) (x : α) :
+  (f * g) x = f x * g x := rfl
```

Comment:
If we're adding this lemma, we should also add `one_apply`, `nat_cast_apply`, and similar for all the other `coe_<foo>` lemmas in this section.